docs/getting-started/github-apps.rst

```diff
@@ -51,15 +46,51 @@ How the Dockstore GitHub App works
 ----------------------------------
 
 With the Dockstore GitHub App, the synchronization is done automatically. When
-you publish a new release of a service on GitHub, Dockstore is notified,
-and Dockstore updates its copy of the service. After you publish a new release
-of a service on GitHub, a new version of the service will be present in
+you publish a new release of a workflow on GitHub, Dockstore is notified,
+and Dockstore updates its copy of the workflow. After you publish a new release
+of a workflow on GitHub, a new version of the workflow will be present in
 Dockstore shortly afterwards.
 
+For this to work, a ``/.dockstore.yml`` file is required in each GitHub repository you want
+to associate with a workflow on Dockstore. A template for both workflows and services are shown below,
+along with explanations for each field. For every branch on GitHub that has one of these files, a corresponding entry
+will be made on Dockstore.
+
+Workflow YML File
+++++++++++++++++++
+
+.. code:: yaml
+
+   version: 1.2
+   workflows:
+      - name: aligner
+        subclass: cwl
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+        - /test/dockstore.cwl.json
+
+version
+    The version of the .dockstore.yml file. Currently at 1.2.
```

Comment:
How about the "The version of the .dockstore.yml schema..." or "The .dockstore.yml schema version". I could see people getting slightly confused with this and their own versions otherwise.